Revit_Core_Engine/Query/ElementsOwnedByOtherUsers.cs

```diff
@@ -0,0 +1,55 @@
+/*
+ * This file is part of the Buildings and Habitats object Model (BHoM)
+ * Copyright (c) 2015 - 2022, the respective contributors. All rights reserved.
+ *
+ * Each contributor holds copyright over their respective contributions.
+ * The project versioning (Git) records all such contribution source information.
+ *                                           
+ *                                                                              
+ * The BHoM is free software: you can redistribute it and/or modify         
+ * it under the terms of the GNU Lesser General Public License as published by  
+ * the Free Software Foundation, either version 3.0 of the License, or          
+ * (at your option) any later version.                                          
+ *                                                                              
+ * The BHoM is distributed in the hope that it will be useful,              
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of               
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the                 
+ * GNU Lesser General Public License for more details.                          
+ *                                                                            
+ * You should have received a copy of the GNU Lesser General Public License     
+ * along with this code. If not, see <https://www.gnu.org/licenses/lgpl-3.0.html>.      
+ */
+
+using Autodesk.Revit.DB;
+using Autodesk.Revit.DB.Analysis;
+using Autodesk.Revit.UI;
+using BH.Engine.Adapters.Revit;
+using BH.oM.Base;
+using BH.oM.Base.Attributes;
+using System.Collections.Generic;
+using System.ComponentModel;
+using System.Linq;
+
+namespace BH.Revit.Engine.Core
+{
+    public static partial class Query
+    {
+        /***************************************************/
+        /****              Public methods               ****/
+        /***************************************************/
+        [Description("Returns a sublist of elements, from the given list of elements, that are owned by other users.")]
+        [Input("elements", "Revit elements to query for its checkout status.")]
+        [Output("elementsOwnedByUsers", "List of elements that are owned by other users.")]
+        public static List<Element> ElementsOwnedByOtherUsers(this List<Element> elements)
+        {
+            if (elements == null || elements.Count <= 0)
+            {
+                BH.Engine.Base.Compute.RecordError("Element list cannot be null or empty.");
+                return null;
+            }
```

Comment:
Actually, there is nothing wrong in empty lists - they should pass and return empty lists again, otherwise your code will keep on breaking every time you pass an empty list (which is a perfectly usual case).

```suggestion
            if (elements == null)
            {
                BH.Engine.Base.Compute.RecordError("Element list cannot be null.");
                return null;
            }
```

**Note:** this comment refers to all cases where you do such check.